README.md

```diff
@@ -18,19 +25,25 @@ Clone the repo of [*mini-lab*](https://github.com/metal-stack/mini-lab) and *xcl
 └── xcluster
 ```
 
-Download the prerequisite of [*mini-lab*](https://github.com/metal-stack/mini-lab#requirements). Then,
+COMMENT: explain mini-lab briefly, most of the people will not know what it is.
+
+Download the prerequisites of [*mini-lab*](https://github.com/metal-stack/mini-lab#requirements). Then,
 
 ```bash
 cd mini-lab
 make
 ```
 
-It's going to take some time to finish. From time to tiem, do
+It's going to take some time to finish. Behind the scene, a [kind](https://github.com/kubernetes-sigs/kind/) cluster is created, *metal-api* related kubernetes resources are deployed, and multiple *linux kernel-based virtual machines* are created for *metal-stack* switches and machines.
+
+From time to time, do
```

Comment:
"From time to time, do ... until two waiting machines are shown" --> add console output how this looks like

Reply:
It's already there. I will clean up the comments later.
```console
ID                                          LAST EVENT   WHEN     AGE  HOSTNAME  PROJECT  SIZE          IMAGE  PARTITION
e0ab02d2-27cd-5a5e-8efc-080ba80cf258        Waiting      8s                               v1-small-x86         vagrant
2294c949-88f6-5390-8154-fa53d93a3313        Waiting      8s                               v1-small-x86         vagrant
```